include/class.config.php

```diff
@@ -207,6 +207,7 @@ class OsticketConfig extends Config {
         'agent_avatar' => 'gravatar.mm',
         'ticket_lock' => 2, // Lock on activity
         'max_open_tickets' => 0,
+        'files_req_auth' => 0,
```

Comment:
I think we should default to requiring authentication, lockdown by default seems to be the expected norm nowadays.